src/Microsoft.ML.FastTree/doc.xml

```diff
@@ -24,7 +24,7 @@
           The output of the ensemble produced by MART on a given instance is the sum of the tree outputs.
         </para>
         <list type='bullet'>
-          <item><description>In case of a binary classification problem, the output is converted to a probability by using some form of calibration.</description></item>
```

Comment:
>case [](start = 32, length = 4)

is the deletion of word `case` by mistake?

Reply:
good catch

---
In reply to: [265813438](https://github.com/dotnet/machinelearning/pull/2970#discussion_r265813438) [](ancestors = 265813438)